.mergify.yml.jinja

```diff
@@ -0,0 +1,13 @@
+pull_request_rules:
+    - name: automatically merge approved PRs with the ready-to-merge label
+      conditions:
+          {%- for status in required_status %}
+          - "status-success={{status}}"
+          {%- endfor %}
+          - "#approved-reviews-by>=1"
+          - label=ready-to-merge
```

Comment:
Will mergify still respect the github required checks? Or if all these are met, will it just merge anyway?

Reply:
Mergify has never respected the github required checks